frontend/src/sagas.js

```diff
@@ -2395,6 +2329,50 @@ export function* closeWorkflowItemSaga({ projectId, subprojectId, workflowitemId
     });
   }, showLoading);
 }
+export function* rejectWorkflowItemSaga({ projectId, subprojectId, workflowitemId,showLoading }) {
+  yield execute(function* () {
+    const confirmed = yield select(getConfirmedState);
+    const rejectReason = yield select(getWorkflowitemReject);    
+    if (confirmed !== true) {
```

Comment:
Do not fetch rejectReason directly. Instead, move it to the paramater list in line 2332.

See assignProjectSaga for reference. There, the assigneeDisplayName is also forwarded through the paramater list